drivers/virtualbox/vm.go

```diff
@@ -0,0 +1,55 @@
+package virtualbox
+
+import (
+	"bufio"
+	"io"
+	"strconv"
+	"strings"
+)
+
+type VirtualBoxVM struct {
+	CPUs   int
+	Memory int
+}
+
+func parseVMInfo(r io.Reader) (*VirtualBoxVM, error) {
+	s := bufio.NewScanner(r)
+	vm := &VirtualBoxVM{}
+	for s.Scan() {
+		line := s.Text()
```

Comment:
The bit after `line := s.Text()`.  Should work fine to just pass `line` into it right?